django/test/utils.py

```diff
@@ -504,6 +504,16 @@ def extend_sys_path(*paths):
 
 
 @contextmanager
+def clear_lru_cache(lru_cache_object):
```

Comment:
Consider `isolate_lru_cache` -- that seems to make it clearer that it clears both on enter and exit. Don't mind if it's not changed though.